llx/builtin_ip.go

```diff
@@ -59,6 +59,23 @@ func NewIP(s string) IP {
 
 var bitmasks = []byte{0x00, 0x80, 0xc0, 0xe0, 0xf0, 0xf8, 0xfc, 0xfe, 0xff}
 
+func int2ip(i int64) string {
+	cur := i
+
+	d := byte(cur % 256)
```

Comment:
Interested to know why you used modulus (`% 256`) Instead of bitwise and (`& 0xFF`)

Reply:
it aligns a bit more with the descriptive approach of calculating the IP I found, but bitwise op would work just as well, I wonder if it's possibly faster in the compiler :thinking: